setup.py

```diff
@@ -18,7 +19,7 @@
 from setuptools import setup, find_packages
 
 def read(fname):
-    return open(os.path.join(os.path.dirname(__file__), fname)).read()
+    return io.open(os.path.join(os.path.dirname(__file__), fname), encoding='utf-8').read()
```

Comment:
OK. I don't know much about the `io` module, but as long as all tests pass and it solves your cp1251 problems, it's good enough :)